### ISSUE TITLE
Error in reading your fastafile, at position -1. Blank name.

### ISSUE THREAD

**Issue by Leslieogo** [mothur/mothur#529]
Hi,
the fasta file created by the make.contigs command seems be wrong.
Impossible to use it in another command. I have always the sae error message.
Can you help me, please ?
See message in the extract of the log file.

mothur > 
summary.seqs(fasta=samples.trim.contigs.fasta)

Using 8 processors.
Error in reading your fastafile, at position -1. Blank name.
[ERROR]: process 0 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 1 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 2 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 3 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 4 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 5 only processed 1 of 932614 sequences assigned to it, quitting. 
[ERROR]: process 6 only processed 1 of 932614 sequences assigned to it, quitting. 

Error in reading your fastafile, at position -1. Blank name.

**Comment by Leslieogo** [comment]
Here the log file
[mothur.1538204846.zip](https://github.com/mothur/mothur/files/2437316/mothur.1538204846.zip)
The fasta file is too large for post it. I try to find another way to provide you the file.

**Comment by Leslieogo** [comment]
A link to download the fasta file
https://owncloud.list.lu/index.php/s/gFJFhh9WaR0puh1

**Comment by mothur-westcott** [comment]
The fasta link failed to decompress, so I am unable to check the fasta file. I noticed you are running a version of mothur that is over 3 years old. Could you try our latest release? https://github.com/mothur/mothur/releases

**Comment by Leslieogo** [comment]
Yes, I have already try with the lastest version of mothur software (1.35.1). I received exactly the same error message.
A new link for the no compressed file
https://owncloud.list.lu/index.php/s/IMzuZiVYt9KENQy

**Comment by pschloss** [comment]
The most recent version of mothur is v.1.40.5. The version you are using is quite old.

**Comment by mothur-westcott** [comment]
I am not seeing any errors with your file in our current version.

mothur > summary.seqs(fasta=samples.trim.contigs.fasta)

Using 4 processors.

		Start	End	NBases	Ambigs	Polymer	NumSeqs
Minimum:	1	35	35	0	2	1
2.5%-tile:	1	50	50	0	6	186523
25%-tile:	1	368	368	0	7	1865229
Median: 	1	368	368	0	7	3730457
75%-tile:	1	368	368	0	7	5595685
97.5%-tile:	1	462	462	16	35	7274391
Maximum:	1	502	502	91	250	7460913
Mean:	1	360	360	0	8
# of Seqs:	7460913

It took 60 secs to summarize 7460913 sequences.

Output File Names:
samples.trim.contigs.summary

**Comment by Leslieogo** [comment]
So, great good news for my fasta file.
Can you provide me the link for download the version 1.40.5, please ?
Thanks a lot for your help.
